encoder/libav_encoder.cpp

```diff
@@ -630,6 +704,8 @@ void LibAvEncoder::audioThread()
 		av_frame_unref(in_frame);
 		av_packet_unref(in_pkt);
 
+		std::this_thread::yield(); // Added to try to reduce thread load on proccessor
+
```

Comment:
Just a bit intrigued why this is necessary... do we know anything more?

Reply:
@naushir requested this to be added
Seems to put the thread to sleep whilst we wait for a frame as oppose to constantly looping

Reply:
You can remove this for now - I'm not 100% sure if it's needed and we probably want to do a bit more experimenting.